main.py

```diff
@@ -196,23 +194,23 @@ def discussion():
 def clusters(methods=["GET"]):
     start_time = time.time()
     if request.method == "GET":
-        ne_lat = float(request.values['ne_lat'])
-        ne_lng = float(request.values['ne_lng'])
-        sw_lat = float(request.values['sw_lat'])
-        sw_lng = float(request.values['sw_lng'])
-        start_date = datetime.date.fromtimestamp(int(request.values['start_date']))
-        end_date = datetime.date.fromtimestamp(int(request.values['end_date']))
-        fatal = int(request.values['show_fatal'])
-        severe = int(request.values['show_severe'])
-        light = int(request.values['show_light'])
-        inaccurate = int(request.values['show_inaccurate'])
-        zoom = int(request.values['zoom'])
-
-        results = retrieve_clusters(ne_lat, ne_lng, sw_lat, sw_lng,
-                                    start_date, end_date,
-                                    fatal, severe, light, inaccurate, zoom)
-
-        logging.debug('calculating clusters took ' + str(time.time() - start_time))
+        kwargs = {'ne_lat': float, 'ne_lng': float, 'sw_lat': float, 'sw_lng': float, 'zoom': int, 'show_fatal': bool,
+                  'show_severe': bool, 'show_light': bool, 'approx': bool, 'accurate': bool, 'show_markers': bool,
+                  'show_discussions': bool, 'show_urban': int, 'show_intersection': int, 'show_lane': int,
+                  'show_day': int, 'show_holiday': int, 'show_time': int, 'start_time': int, 'end_time': int,
+                  'weather': int, 'road': int, 'separation': int, 'surface': int, 'acctype': int,
+                  'controlmeasure': int, 'district': int}
+
+        for arg in kwargs:
```

Comment:
Just don't call it `kwargs`... it's not a meaningful name anyway. And constants should be in all caps.